### ISSUE TITLE
4.1.1 error on system install: Invalid map_template instance: "caverns_of_chaincolt_gates".

### ISSUE THREAD

**Issue by akien-mga** [Andrettin/Wyrmsun#183]
I updated my Mageia package to 4.1.1 locally and things now build fine/the engine runs properly, but there seems to be an issue with the data:

```
$ wyrmsun 
Stratagus v4.1.1, Copyright (c) 1998-2020 by The Stratagus Project
  written by Lutz Sammer, Fabrice Rossi, Vladi Shabanski, Patrice Fortier,
  Jon Gabrielson, Andreas Arens, Nehal Mistry, Jimmy Salmon, Pali Rohar,
  cybermind, Andrettin and others.
        https://github.com/Andrettin/Wyrmgus
Compile options ZLIB OPENGL LINUX X11 

Stratagus may be copied only under the terms of the GNU General Public License
which may be found in the Stratagus source kit.

DISCLAIMER:
This software is provided as-is.  The author(s) can not be held liable for any
damage that might arise from the use of this software.
Use it at your own risk.

Invalid map_template instance: "caverns_of_chaincolt_gates".
Error parsing config file "/usr/share/games/wyrmsun/data/map_templates/nidavellir/untersberg.cfg".
Thanks for playing Stratagus.
```

```
$ ls -l /usr/share/games/wyrmsun/data/map_templates/nidavellir/untersberg.cfg
-rw-r--r-- 1 root root 1465 Jan  9 22:56 /usr/share/games/wyrmsun/data/map_templates/nidavellir/untersberg.cfg
```

The file seems properly formatted, not sure what the issue is.

**Comment by Andrettin** [comment]
This commit should solve that:
https://github.com/Andrettin/Wyrmsun/commit/c3d822e9b7b89e8132b71cecd41edb0f3e8c9674

I had forgotten to add the new "modules" folder to the directory list in CMakeLists.txt.

**Comment by akien-mga** [comment]
Seems fixed in 4.1.2 indeed, but now I have another issue #184.
